fixed incositency while changing from light to dark vice versa

abhayymishraa opened the pull request:
<!-- Thanks for contributing to OWASP Nest!-->

<!-- Don't forget to link your PR to an existing issue if any.-->
Resolves #662 


<!-- Describe the big picture of your changes.-->
Add the PR description here.

Tasks
- [x] fixed incosistency when switching from dark to light  or light to dark

### Preview


[Screencast from 30-01-25 11:45:08 PM IST.webm](https://github.com/user-attachments/assets/18a349ad-06f9-4989-868b-312f452eb721)


<!-- Thanks again for your contribution!-->

arkid15r reviewed (commented):
@Rajgupta36 could you verify the bug fix?
Thank you

abhayymishraa commented:
@Naveen-Pal  i will fix it !! i somehow missed that

Naveen-Pal commented:
![image](https://github.com/user-attachments/assets/c25a5e52-75e7-43b3-a6da-80cc41a5a09f)
Also pagination component also look like to have a slight delay.

abhayymishraa commented:
@Naveen-Pal  It would be like this. Is it good. could you review it ?
[Screencast from 31-01-25 12:51:23 AM IST.webm](https://github.com/user-attachments/assets/9637b8a5-ffa8-4d6a-8436-819cc04cb3ba)

Naveen-Pal commented:
Looks good

arkid15r reviewed (changes requested):
@abhayymishraa this PR contains 0 files changed.

abhayymishraa commented:
> @abhayymishraa this PR contains 0 files changed.

hey @arkid15r  because that already got merged by you from my different account remember :)

abhayymishraa commented:
hey @arkid15r  this is it can you complete this request so that it can be mentioned in the tasks 
I have added the pre-commit to check test files. 
i have fixed the inconsistency.

arkid15r commented:
> hey @arkid15r this is it can you complete this request so that it can be mentioned in the tasks I have added the pre-commit to check test files. i have fixed the inconsistency.

I believe it's tracked here -- https://github.com/OWASP/Nest/issues/662
